scripts/verify_extract_sdk.py

```diff
@@ -107,8 +107,8 @@ def replace_models(sdk_root, project_root):
     update_models_dir = os.path.join(sdk_root, 'Models')
     
     print("Moving new models into project at {0}".format(project_models_dir))
-    shutil.rmtree(project_models_dir)
-    shutil.move(update_models_dir, project_models_dir)
+    #shutil.rmtree(project_models_dir)
+    #shutil.move(update_models_dir, project_models_dir)
```

Comment:
Is this intentionally disabled?

Reply:
Yes, they are. Actually causing problem with update script. I made temporary fixes to the script but I will go further and make the script working as it originally was supposed to.